Installation/Mapping/XxllncResultaattypeQueryInMapping.json

```diff
@@ -0,0 +1,16 @@
+{
+  "title": "XxllncResultaattypeQueryInMapping",
+  "$id": "https://development.zaaksysteem.nl/mapping/xxllnc.XxllncResultaattypeQueryInMapping.mapping.json",
+  "$schema": "https://docs.commongateway.nl/schemas/Mapping.schema.json",
+  "version": "0.0.1",
+  "passTrough": true,
+  "mapping": {
+    "zaaktypeIdentificatie": "zaaktype_identificatie",
+    "beginGeldigheid": "{% if datumGeldigheid|default %}datumGeldigheid{% endif %}"
```

Comment:
Shouldn't this be datum_geldigheid (looking at the casts?)

Reply:
Or the other way around 🤔. Should the cast be beginGeldigheid